app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -384,19 +426,21 @@ fun PocketStoriesCategories(
  * @param modifier [Modifier] to be applied to the layout.
  * @param textColor [Color] to be applied to the text.
  */
+@OptIn(ExperimentalComposeUiApi::class) // for testTagsAsResourceId
 @Composable
 fun PoweredByPocketHeader(
     onLearnMoreClicked: (String) -> Unit,
     modifier: Modifier = Modifier,
     textColor: Color = FirefoxTheme.colors.textPrimary,
 ) {
-    val link = stringResource(R.string.pocket_stories_feature_learn_more)
-    val text = stringResource(R.string.pocket_stories_feature_caption, link)
-    val linkStartIndex = text.indexOf(link)
-    val linkEndIndex = linkStartIndex + link.length
-
     Column(
-        modifier = modifier,
+        modifier = modifier
+            .then(
+                Modifier.semantics {
+                    testTagsAsResourceId = true
+                    testTag = "pocket.header"
+                }
+            ),
```

Comment:
nit: the `then` shouldn't be necessary here
```suggestion
        modifier = modifier.semantics {
            testTagsAsResourceId = true
            testTag = "pocket.header"
        },
```